tests/unit/utils/serialize/test_python.py

```diff
@@ -0,0 +1,45 @@
+import pytest
```

Comment:
The code part LGTM.

`ast.literal_eval` are quite stable. I don't consider it to be worthy to do a detailed test on it. But here the tests involve collections and sequences. I just wondered that we don't test them in previous?

Reply:
We test them here: 

https://github.com/iterative/dvc/blob/master/tests/unit/dependency/test_params.py#L114

I went for a separate unit test as that one involves `ParamsDependency` and `Stage` in the mix. 

I could remove this file and just add a test case for the `UNARY_OP` in the file linked above.

Reply:
>  I don't consider it to be worthy to do a detailed test on it.

It's not testing `ast.literal_eval`, it's testing that we are converting assignment statement into a key-value pair correctly.

Reply:
> > I don't consider it to be worthy to do a detailed test on it.
> 
> It's not testing `ast.literal_eval`, it's testing that we are converting assignment statement into a key-value pair correctly.

Yes, I know that ` But here the tests involve collections and sequences. I just wondered that we don't test them in previous?`